### ISSUE TITLE
How we can use PerceptualSimilarity loss function for images with different sizes

### ISSUE THREAD

**Issue by kiashann** [richzhang/PerceptualSimilarity#104]
Hi, I want to use this loss function for two images with different size. How I can do that?
It's possible to use this loss function for predicted images with different sizes with input images?
@dvschultz

**Comment by njuaplusplus** [comment]
I used [PyTorch resize function](https://pytorch.org/vision/stable/generated/torchvision.transforms.functional.resize.html#torchvision.transforms.functional.resize) to resize them to the same size.
